cmake/modules/RootBuildOptions.cmake

```diff
@@ -145,18 +145,18 @@ ROOT_BUILD_OPTION(python3 OFF "Python 3 ROOT bindings, requires python >= 3.1")
 ROOT_BUILD_OPTION(qt OFF "Qt graphics backend, requires libqt >= 4.8")
 ROOT_BUILD_OPTION(qtgsi OFF "GSI's Qt integration, requires libqt >= 4.8")
 ROOT_BUILD_OPTION(r OFF "R ROOT bindings, requires R, Rcpp and RInside")
-ROOT_BUILD_OPTION(rfio ON "RFIO support, requires libshift from CASTOR >= 1.5.2")
+ROOT_BUILD_OPTION(rfio OFF "RFIO support, requires libshift from CASTOR >= 1.5.2")
 ROOT_BUILD_OPTION(roofit OFF "Build the libRooFit advanced fitting package")
 ROOT_BUILD_OPTION(root7 OFF "Build the ROOT 7 interface prototype, requires >= cxx14")
 ROOT_BUILD_OPTION(rpath OFF "Set run-time library load path on executables and shared libraries (at installation area)")
 ROOT_BUILD_OPTION(ruby OFF "Ruby ROOT bindings, requires ruby >= 1.8")
 ROOT_BUILD_OPTION(runtime_cxxmodules OFF "Enable runtime support for C++ modules.")
-ROOT_BUILD_OPTION(sapdb ON "MaxDB/SapDB support, requires libsqlod and libsqlrte")
+ROOT_BUILD_OPTION(sapdb OFF "MaxDB/SapDB support, requires libsqlod and libsqlrte")
 ROOT_BUILD_OPTION(shadowpw ON "Shadow password support")
```

Comment:
What does this option do?

Reply:
I *think* this could be related to HTTP server and friends? @linev @bellenot

Reply:
No, THttpServer  does not use any external libraries

Reply:
rpdutils is using it. It's really a config flag, not a module. @gganis - what's your recommendation?

Reply:
It turns on support for reading/using the  unix shadow password in/for proof[serv]

Reply:
Actually was only used by rootd/proofd daemons via rpdutils (and the cmake check only tests if /etc/shadow is there and issues a warning on non linux machines ...). Now, being rootd/proofd deprecated I think we should disable all the authentication related things. All that stuff is unused (unless is used by THttpServer, @linev?).

Reply:
No, THttpServer and civetweb does not have nay external dependencies (beside librt.so)
There is no plans to include any kind of authentications

Reply:
Ok - this is then a candidate for being OFF. Of course, this is not irreversible (as any of the other changes in this PR!)